packages/web-components/src/components/ic-button/ic-button.tsx

```diff
@@ -553,13 +553,13 @@ export class Button {
     return (
       <Host
         class={{
-          ["disabled"]: this.disabled && !this.loading,
+          ["ic-button-disabled"]: this.disabled && !this.loading,
           [`button-variant-${this.variant}`]: true,
           [`button-size-${this.size}`]: true,
```

Comment:
button variant and size haven't been prefixed, what's the thinking there?

Reply:
I was just updating ones that were generic, but I do agree these should be prefixed with ic so it matches the others. Will update